Ponca/src/Fitting/unorientedSphereFit.hpp

```diff
@@ -102,21 +56,18 @@ UnorientedSphereFitImpl<DataPoint, _WFunctor, T>::finalize ()
     // 1. finalize sphere fitting
     Scalar invSumW = Scalar(1.) / Base::getWeightSum();
 
+    m_matQ.template topLeftCorner<Dim,Dim>().setIdentity();
+    m_matQ.col(Dim).template head<Dim>() = Base::m_sumP * invSumW;
+    m_matQ.row(Dim).template head<Dim>() = Base::m_sumP * invSumW;
+    m_matQ(Dim,Dim) = m_sumDotPP * invSumW;
 
-    MatrixBB Q;
-    Q.template topLeftCorner<Dim,Dim>().setIdentity();
-    Q.col(Dim).template head<Dim>() = Base::m_sumP * invSumW;
-    Q.row(Dim).template head<Dim>() = Base::m_sumP * invSumW;
-    Q(Dim,Dim) = m_sumDotPP * invSumW;
-    m_matA *= invSumW;
```

Comment:
Yes, I removed it since it does not change the resulting fitted sphere, and this normalization does not appear in the original paper (eq. 6 of https://inria.hal.science/hal-00857265/PDF/MLS_grad_2013.pdf). 
I can put it back to be completely sure to have the same behavior as previously though (in case there is some numerical imprecision).